changes/7685.feature

```diff
@@ -0,0 +1,3 @@
+Add htmx webasset to CKAN. For more information check the `documentation <https://docs.ckan.org/extensions/htmx.html>`_.
```

Comment:
This URL is 404 Not Found for me.

Reply:
Yes @ThrawnCA because it is a new documentation page that only exist in this PR :)

Reply:
I'll try to add a relative link., though I'm not sure if they are available in the CHANGELOG files